satpy/tests/reader_tests/test_fci_l1c_fdhsi.py

```diff
@@ -483,6 +473,34 @@ def test_excs(self, reader_configs):
                     make_dataid(name="ir_123", calibration="unknown"),
                     {"units": "unknown"})
 
+    def test_area_definition_computation(self, reader_configs):
+        """Test that the geolocation computation is correct."""
+        from satpy.readers import load_reader
+
+        filenames = [
+            "W_XX-EUMETSAT-Darmstadt,IMG+SAT,MTI1+FCI-1C-RRAD-FDHSI-FD--"
+            "CHK-BODY--L2P-NC4E_C_EUMT_20170410114434_GTT_DEV_"
+            "20170410113925_20170410113934_N__C_0070_0067.nc",
+        ]
+
+        reader = load_reader(reader_configs)
+        loadables = reader.select_files_from_pathnames(filenames)
+        reader.create_filehandlers(loadables)
+        res = reader.load(['ir_105', 'vis_06'], pad_data=False)
+
+        # test that area_ids are harmonisation-conform <platform>_<instrument>_<service>_<resolution>
+        assert res['vis_06'].attrs['area'].area_id == 'mtg_fci_fdss_1km'
+        assert res['ir_105'].attrs['area'].area_id == 'mtg_fci_fdss_2km'
+
+        # test area extents computation
+        np.testing.assert_array_almost_equal(np.array(res['ir_105'].attrs['area'].area_extent),
+                                             np.array([-5568062.23065902, 5168057.7600648,
+                                                       16704186.692027, 5568062.23065902]))
+
+        # check the projection string
+        expected_proj_str = '+ellps=WGS84 +h=35786400 +lon_0=0 +no_defs +proj=geos +type=crs +units=m +x_0=0 +y_0=0'
+        assert res['ir_105'].attrs['area'].proj_str == expected_proj_str
```

Comment:
I'm not convinced the proj string should be used as reference (and pyproj seems to hint at this in a warning everytime we use the proj string). How about checking the individual CRS attributes instead? or maybe the wkt text instead?

Reply:
Hi Martin, thanks for your review! Good point, I switched to checking the CRS attributes!